jetty-websocket/websocket-util/src/main/java/module-info.java

```diff
@@ -21,8 +21,8 @@
     exports org.eclipse.jetty.websocket.util;
     exports org.eclipse.jetty.websocket.util.messages;
 
-    requires org.eclipse.jetty.util;
     requires org.slf4j;
-    requires org.eclipse.jetty.io;
+    requires transitive org.eclipse.jetty.io;
+    requires transitive org.eclipse.jetty.util;
     requires transitive org.eclipse.jetty.websocket.core.common;
```

Comment:
Because you `requires transitive` `core.common`, you won't need to `requires transitive` again `.io` and `.util`